certbot/tests/main_test.py

```diff
@@ -519,7 +519,8 @@ def test_update_symlinks(self, mock_cert_manager):
         self.assertEqual(1, mock_cert_manager.call_count)
 
     @mock.patch('certbot.cert_manager.certificates')
-    def test_certificates(self, mock_cert_manager):
+    @test_util.patch_get_utility()
```

Comment:
I'm assuming you modified the 4 methods in this file because they are the ones that previously printed to `stdout`? If it's not too difficult, it'd be nice if we could modify the `_call_no_clientmock` method which should result in fewer code changes and future methods can benefit without knowing the intricacies here.

If making this change causes a few test failures, perhaps we could make a method called `_call_no_get_utility_mock` that works like `_call_no_clientmock`. If making this change causes a lot of test failures though, I think the approach you took here is perfectly fine.